lib/hanami/commands/console.rb

```diff
@@ -22,7 +22,7 @@ def reload!
         'irb'  => 'IRB'
       }.freeze
 
-      DEFAULT_ENGINE = 'irb'.freeze
+      DEFAULT_ENGINE = ['irb'].freeze
```

Comment:
@railsme I'd leave that hash as is. Because using `if` is just slightly slower than `array` way but with the gain of readability, I'll wait for @jodosha to weigh in his opinion